external/src/main/java/com/redhat/cloud/policies/engine/history/PoliciesHistoryEntry.java

```diff
@@ -0,0 +1,97 @@
+package com.redhat.cloud.policies.engine.history;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "policies_history")
+public class PoliciesHistoryEntry {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    private String tenantId;
```

Comment:
Same comment as in the `policies-ui-backend`, should we call this `accountId` ? `tenant` is a term used by hawkular-alerts, akin to say `triggerId` instead of `policyId`.

Reply:
Something else is that in `policies-ui-backend`, the fields are annotated with e.g. `@Column(name = "tenant_id")`, but not on `policies-engine`

Reply:
> Same comment as in the `policies-ui-backend`, should we call this `accountId` ? `tenant` is a term used by hawkular-alerts, akin to say `triggerId` instead of `policyId`.

Same kind of reply than in policies-ui-backend: `tenantId` is everywhere there 😃

Reply:
> Something else is that in `policies-ui-backend`, the fields are annotated with e.g. `@Column(name = "tenant_id")`, but not on `policies-engine`

See [this](https://github.com/RedHatInsights/policies-engine/pull/207#discussion_r675075264).